tests/testthat/test-profiling.R

```diff
@@ -4,7 +4,6 @@ if (not_on_cran()) {
 
 test_that("profiling works if profiling data is present", {
   skip_on_cran()
-  skip_if(cmdstan_version() < "2.26.0")
```

Comment:
I removed this skips as we always tests with the latest version and I think this is not needed actually.